src/lando/api/legacy/workers/base.py

```diff
@@ -178,7 +199,6 @@ def loop(self):
         if self.last_job_finished is False:
             logger.info("Last job did not complete, sleeping.")
             self.throttle(self.worker_instance.sleep_seconds)
-            self.refresh_active_repos()
```

Comment:
This was a left-over duplicate from an incorrect conflict resolution. It's already done just above.

Reply:
Oh, right. This is a refresh after tho throttle. Makes sense.